_template/index.html

```diff
@@ -4,7 +4,7 @@
         <meta charset="UTF-8">
         <meta name="viewport" content="width=device-width">
 
-        <title v-text="project_name ? project_name + ' | Pattern Library' : 'Pattern Library'"></title>
+        <title v-text="project_name ? project_name + ' | ' + (theme.titles.library_title ? theme.titles.library_title : 'Pattern Library') : 'Pattern Library'"></title>
```

Comment:
Hey @hankchizljaw, it looks to me as if one edge cases is still missing: 
`project_name`is null and a custom `theme.titles.library_title` would fall back to 'Pattern Library' according to the ternary operator, I will post a Pull Request on your branch shortly. 

Otherwise everything works as expected.